### ISSUE TITLE
Created/last modified label for saved login off center in landscape orientation on devices with notch

### ISSUE THREAD

**Issue by michaelbiv** [mozilla-mobile/firefox-ios#9242]
## Steps to reproduce
* Requires iPhone with notch
1. Tap the menu button
2. Tap Passwords
3. Tap a saved login
4. Rotate device into landscape orientation

### Expected behavior
- Created/modified date label should remain centered horizontally in table view cell

### Actual behavior
- Created/modified date label shift to the right and appear off-center

### Device & build information
* Device: iPhone 12 Pro Max
* Simulators: iPhone 13 Pro, iPhone 13 Pro Max
* Build version: [85d8ceb](https://github.com/mozilla-mobile/firefox-ios/commit/85d8ceb33321b0697157357706eecbd4f66812fa)

### Notes
Attachments: 

Centered in portrait:
![portrait](https://user-images.githubusercontent.com/829610/135790941-fa7e46d2-9424-4b2a-b726-78ce71d2ad1b.png)

Off-center in landscape:
![landscape](https://user-images.githubusercontent.com/829610/135790962-1b11f3b7-2aad-4447-80e2-c58a87c979d2.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-3245)

**Comment by SimonBasca** [comment]
Thanks for reporting the issue. Closing as duplicate of #8876

**Comment by SimonBasca** [comment]
Sorry for closing the issue, just noticed it has a PR attached. Closing #8876 in favor of this one.

**Comment by michaelbiv** [comment]
@SimonBasca No problem.  I submitted two separate pull requests which are somewhat related.  One was for the text being cutoff (#9239 - utilizes a right inset as opposed to left) and a second for the text being off-center in landscape (#9242 - subtracts right safe area inset).  

I submitted a third pull request as well (#9240) which addresses the separators not being hidden or full as desired on initial load.  Also related to backgrounding and bringing back to foreground.  I can combine any of these if needed as well.
